src/omnisharp/OmnisharpPackageCreator.ts

```diff
@@ -55,4 +55,13 @@ function GetPackageFromArchitecture(inputPackage: Package, serverUrl: string, ve
     };
 
     return versionPackage;
-}
\ No newline at end of file
+}
+
+export function GetVersionFilePackage(serverUrl: string, pathInServer: string): Package {
```

Comment:
This is confusing. Passing around `packages` to represent the actual packages we download, sure. Creating a `package` to represent downloading a text file...strange. Can we at least add a comment explaining why?

Reply:
let's add the comment for now, then refactor more in a future PR? all of the great work that @DustinCampbell did around proxy negotiation, etc. is currently best accessed via the Package construct.

Reply:
Works for me.